WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -335,7 +335,7 @@ private static Money FindChange(IEnumerable<Output> decomposition, HashSet<Outpu
 		var secondLargestInput = inputEffectiveValues.OrderByDescending(x => x).Skip(1).First();
 		var demonsForBreakDown = Denominations
 			.Where(x => x.EffectiveCost <= secondLargestInput) // Take only affordable denominations.
-			.OrderByDescending(x => x.EffectiveAmount); // If the amount is the same, the cheaper to spend should be the first - so greedy will take that.
```

Comment:
FTR in Sake this was `EffectiveCost` to begin with, so someone - maybe me - has been confused by this already before.